src/grisp_connect_client.erl

```diff
@@ -16,11 +16,19 @@
 -export([handle_message/1]).
 
 -behaviour(gen_statem).
--export([init/1, terminate/3, code_change/4, callback_mode/0, handle_event/4]).
+-export([init/1, terminate/3, code_change/4, callback_mode/0]).
+
+% State Functions
+-export([idle/3]).
+-export([waiting_ip/3]).
+-export([connecting/3]).
+-export([connected/3]).
 
 -include_lib("kernel/include/logger.hrl").
 
 -define(STD_TIMEOUT, 1000).
+-define(HANDLE_COMMON,
+    ?FUNCTION_NAME(T, C, D) -> handle_common(T, C, ?FUNCTION_NAME, D)).
```

Comment:
Wouldn't hurt to have more descriptive names for the arguments here. Erlang doc usually uses `EventType`, `EventContent`, `Data`.

Reply:
A it's copy paste from the docs. Well maybe I don't like how it's done in the docs.